README.md

```diff
@@ -852,6 +852,150 @@ TODO-A: - TBD
 TODO-B: Problems: None
 ```
 
+
+# Re-Reviewing an approved package
+
+Right now this is **not active**, it is - so far - a documentation of the intent to do so helping all involved parties to better understand what this would mean.
+
+The MIR team and thereby in extension Ubuntu continuously evolves the MIR ruling, for example we add rules based on painful lessons learned to avoid the same to happen again. In that regard some asks for testing, quality and such might be best practise now, but have not been fulfilled by packages that are in main for a long time.
+
+The proposed approach would be to:
+
+1. The MIR team would aim for a constant demand of e.g. "1 review per week per MIR member"
+   1. On one hand that means if new requests are assigned in a week that we make no progress on the backlog, but OTOH this keeps effort plannable)
+2. We'd run a trimmed down check which is a subset of our usual check (see below)
+3. As on the initial MIR review we might end up recommending a security (re-)review which goes into their queue at low prio
+
+## Special cases
+Special case - packges in main without an launchpad bug based MIR process.
+Some older cases do not have anything for auditability, for those we'll reach out to the [owning team](http://reqorts.qa.ubuntu.com/reports/m-r-package-team-mapping.html) and ask to file an MIR as they should ask (and answer) themselfe the same questions we do on a modern MIR.
```

Comment:
```suggestion
Some older cases do not have anything for auditability. For those packages we'll reach out to the [owning team](http://reqorts.qa.ubuntu.com/reports/m-r-package-team-mapping.html) and ask them to file an MIR as they should ask (and answer) themselfe the same questions we do on a modern MIR.
```

suggestion: slightly different punctuation/wording for better reading.

Reply:
Integrated (slightly different than proposed)